contracts/base/amm/services/AmmOpenSwapServiceBaseV1.sol

```diff
@@ -24,7 +24,7 @@ abstract contract AmmOpenSwapServiceBaseV1 {
 
     address public constant ETH_ADDRESS = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
 
-    uint256 public immutable version = 1;
+    uint256 public immutable version = 2;
```

Comment:
maybe we can change this to contract address?

Reply:
In router we are not using getVersion